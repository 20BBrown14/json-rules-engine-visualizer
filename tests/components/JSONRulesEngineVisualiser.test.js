import React from 'react';
import JSONRuleEngineVisualiser from '../../src/components/JSONRulesEngineVisualiser';

describe('JSONRuleEngineVisualiser', () => {
  it('renders properly', () => {
    const testComponent = render(<JSONRuleEngineVisualiser />);
    expect(testComponent).toMatchSnapshot();
  });
});
