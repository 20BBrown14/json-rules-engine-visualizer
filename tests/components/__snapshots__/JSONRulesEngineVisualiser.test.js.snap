// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JSONRuleEngineVisualiser renders properly 1`] = `
<div>
  hello world
</div>
`;
